nsqlookupd.c

```diff
@@ -1,19 +1,11 @@
-#include "json.h"
 #include "nsq.h"
-#include "http.h"
```

Comment:
It is better to include "http.h" here, where it only applies to `nsqlookupd.c`, instead of in `nsq.h`, where it applies to all files which `#include "nsq.h"`. This helps for incremental rebuilds, and general  ... well-contained-ness.